tests/framework/e2e/etcd_process.go

```diff
@@ -375,7 +375,11 @@ func (f *BinaryFailpoints) SetupHTTP(ctx context.Context, failpoint, payload str
 	}
 	defer resp.Body.Close()
 	if resp.StatusCode != http.StatusNoContent {
-		return fmt.Errorf("bad status code: %d", resp.StatusCode)
+		errMsg, err := io.ReadAll(resp.Body)
```

Comment:
Currently `gofail PUT` doesn't return any data in the body. So can we keep it unchanged for simplicity?

Reply:
Sure! Rolling back!

Thanks for reviewing!

Reply:
Sorry, my previous comment isn't correct. `gofail PUT` will write some error message when failed (see link below). Not a big deal. But please keep keep it consistent between `SetupHTTP` and `DeactivateHTTP`. 

https://github.com/etcd-io/gofail/blob/93c579a86c468537617dedd1dc0f63c8f27926ca/runtime/http.go#L69-L81

Reply:
Sorry @ahrtr  I pushed the wrong commit at the airport :( 

I checked the gofail source code and saw that both PUT and DELETE will return error messages! Thus the commit will stay as-is.